ui/appui-react/src/appui-react/statusfields/SnapMode.tsx

```diff
@@ -170,8 +208,15 @@ export function SnapModeField(props: SnapModeFieldProps) {
       <Button
         styleType="borderless"
         title={title}
-        // eslint-disable-next-line deprecation/deprecation
-        endIcon={<Icon iconSpec={getIconFromIconName(iconName)} />}
+        endIcon={
+          getSnapCount(snapMode) > 1 ? (
```

Comment:
I think this is unnecessary once we refactor and have the `availableSnapModeEntries`.